src/status_im/contexts/communities/actions/addresses_for_permissions/events.cljs

```diff
@@ -0,0 +1,60 @@
+(ns status-im.contexts.communities.actions.addresses-for-permissions.events
+  (:require
+    [schema.core :as schema]
+    [taoensso.timbre :as log]
+    [utils.re-frame :as rf]))
+
+(defn get-permissioned-balances
+  [{[event-id _] :original-event} [community-id]]
+  {:fx [[:json-rpc/call
+         [{:method     :wakuext_getCommunityPermissionedBalances
+           :params     [{:communityId community-id}]
+           :on-success [:communities/get-permissioned-balances-success community-id]
+           :on-error   (fn [error]
+                         (log/error "failed to get balances"
+                                    {:community-id community-id
+                                     :event        event-id
+                                     :error        error}))}]]]})
+
+(schema/=> get-permissioned-balances
+  [:=>
+   [:catn
+    [:cofx :schema.re-frame/cofx]
+    [:args [:schema [:? [:catn [:community-id :string]]]]]]
+   [:map {:closed true}
+    [:fx
+     [:tuple
+      [:tuple [:= :json-rpc/call] :schema.common/rpc-call]]]]])
+
+(rf/reg-event-fx :communities/get-permissioned-balances get-permissioned-balances)
+
+(defn get-permissioned-balances-success
+  [{:keys [db]} [community-id response]]
+  {:db (assoc-in db [:communities/permissioned-balances community-id] response)})
+
+(def ^:private ?account-address keyword?)
+
+(def ^:private ?permissioned-balances-response
+  [:map-of
+   ?account-address
+   [:sequential
+    [:map {:closed true}
+     [:type int?]
+     [:symbol string?]
+     [:decimals int?]
+     [:amount string?]]]])
+
+(schema/=> get-permissioned-balances-success
+  [:=>
+   [:catn
+    [:cofx :schema.re-frame/cofx]
+    [:args
+     [:schema
+      [:?
+       [:catn
+        [:community-id string?]
+        [:response ?permissioned-balances-response]]]]]]
```

Comment:
Any reason there's a zero-or-one here? args should always be a seq right?

```suggestion
    [:args
    
     [:schema
       [:catn
        [:community-id string?]
        [:response ?permissioned-balances-response]]]]]
```

Reply:
@yqrashawn, I can give this another try. I swear I had some schema error and had to use this trick, similar to https://github.com/status-im/status-mobile/blob/0349b0ed3d772d71a0cfa6451655d18a965227b9/src/status_im/common/universal_links.cljs#L200

But let me check and remove if possible.

Reply:
I saw ghosts. Removed the unnecessary schema. Thanks for spotting that

Reply:
Seems it's still there. Maybe commit got messed up?

Reply:
Yes, something mysterious happened to the code. I'm rebasing against `develop` to make the final tests. Thanks for reminding me.